extractor.py

```diff
@@ -30,7 +30,7 @@ def progressBar(value, endvalue, bar_length=30):
 	arrow = '-' * int(round(percent * bar_length)-1) + '>'
 	spaces = ' ' * (bar_length - len(arrow))
 
-	sys.stdout.write("\rProgress: [{0}] {1}%".format(arrow + spaces, int(round(percent * 100))))
+	sys.stdout.write("\rProgress: [{}] {}%".format(arrow + spaces, int(round(percent * 100))))
```

Comment:
```python
>>> percent = 0.57
>>> '{:.0%}'.format(percent)
'57%'
```
?

Reply:
@42B Thanks for the reminder that that works, I had totally forgotten.

Reply:
@RedDuckss 

No problem!